polyfill/lib/duration.mjs

```diff
@@ -57,17 +57,14 @@ export class Duration {
         break;
       }
       case 'balance': {
-        let deltaDays;
         ({
-          deltaDays,
-          hour: hours,
-          minute: minutes,
-          second: seconds,
-          millisecond: milliseconds,
-          microsecond: microseconds,
-          nanosecond: nanoseconds
-        } = ES.BalanceTime(hours, minutes, seconds, milliseconds, microseconds, nanoseconds));
-        days += deltaDays;
+          hours,
+          minutes,
+          seconds,
+          milliseconds,
+          microseconds,
+          nanoseconds,
+        } = ES.BalanceDuration(0, hours, minutes, seconds, milliseconds, microseconds, nanoseconds, 'hours'));
```

Comment:
Where is `ES.BalanceDuration` defined?

Reply:
Currently, https://github.com/tc39/proposal-temporal/blob/main/polyfill/lib/ecmascript.mjs#L799-L834

Reply:
ok.  I'm slightly confused because last I checked, the behavior of the polyfill did not balance between duration fields.  I don't see how this PR concretely changes the PR behavior.

Reply:
My guess is that when you checked, you probably used Duration.from(), which currently doesn't balance, instead of `new Duration(..., 'balance')`. Duration.from() is _supposed_ to balance, if requested to, which is implemented in #405 for both the spec and the polyfill. If it makes things less confusing, I'm happy to put this on hold and revisit it after #405 is reviewed and merged.

Reply:
Aha! Yep, that makes sense.